kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -128,13 +128,44 @@ def uninstall(
                 f"Release with name {release_name} not found. Could not uninstall app."
             )
 
-    def template(self, release_name: str, chart: str, api_versions: str = "") -> str:
+    def template(
+        self,
+        release_name: str,
+        chart: str,
+        api_versions: str = "",
+        ca_file: str = "",
+        cert_file: str = "",
+    ) -> str:
         """
         Render chart templates locally and display the output.
         """
-        command = ["helm", "template", release_name, chart]
-        if not api_versions == "":
-            command.extend(["--api-versions", api_versions])
+        command = [
+            "helm",
+            "template",
+            release_name,
+            chart,
+        ]
+        if api_versions:
+            command.extend(
+                [
+                    "--api-versions",
+                    api_versions,
+                ]
+            )
+        if ca_file:
+            command.extend(
+                [
+                    "--ca-file",
+                    ca_file,
+                ]
+            )
+        if cert_file:
+            command.extend(
+                [
+                    "--cert-file",
+                    cert_file,
+                ]
+            )
```

Comment:
extract method

Reply:
Done, but I'm not sure if you would like it. It is a bit more general in comparison with the `upgrade_install` enriching method. I prefer it like I wrote it, but I am not sure if it has any advantages or disadvantages.

Reply:
Why not writing it like the `upgrade_install` enrich method?

Reply:
I thought that parameterizing it like that would be cool as it would work for any helm wrapping method with optional flags, but today I saw that actually I have missed some details. I will rework it.